Modified nav-bar

shobhit2020 opened the pull request:
I just tried to improvise the navigation bar, also it looks fine at the top.
Added a home button, in-page link for footer was not working, fixed that.

I'm thinking to work on canvas, if I can add. But I'm just a beginner, trying to perform.
Thank you

shobhit2020 committed:
Modified nav-bar

Ayush7-BIT commented:
@shobhit2020  can you send me a screenshot of what changes you have made in navbar?

shobhit2020 commented:
> @shobhit2020 can you send me a screenshot of what changes you have made in navbar?

![image](https://user-images.githubusercontent.com/63972729/102708212-52340c80-42c7-11eb-8e47-dd84c2ece889.png)

I fixed the nav bar so that it flows upon every page of the website, added a home button ( clicking on that, will take you to the first page ), rearranged the list items of the nav bar and the on-page link for contact was not working, I fixed that.

Please tell if you're able to see the screen-shot.
